### PR TITLE
feat: add toggle preview json to copy to manifest.json

### PR DESCRIPTION
**Pull request by alexalannunes**
I think that it's more easy to show a json snippet to developer can copy!

Demo

### CODE REVIEW

File: editor.html [hunk 1]
```diff
@@ -297,6 +309,12 @@ <h2 id="modalTitle">Export</h2>
             </label>
           </div>
 
+          <div>
+            <a href="#" class="dialog-action-link" id="showPreviewArrayIcons">Show JSON</a>
```
Comment (NotWoods) [inline editor.html:313]: I'd like to use the `<details>`/`<summary>` elements instead to do the open/close without Javascript.

File: src/editor/export.js [hunk 2]
```diff
@@ -7,8 +7,29 @@ const sizeInputs = /** @type {NodeListOf<HTMLInputElement>} */ (document.getElem
...
+  e.preventDefault();
+  const filteredJson = sizesArr
+    .filter((i) => i)
+    .map((size) => ({
+      src: `maskable_icon_x${size}.png`,
+      type: 'image/png',
+      sizes: `${size}x${size}`,
+    }));
+
+  outputPreview.textContent = JSON.stringify(filteredJson, null, 2);
```
Comment (NotWoods) [inline src/editor/export.js:29]: Updating the text is cheap enough that we can just change it everytime the selected checkboxes are changed. That also removes the need for a separate array, you can read `sizeInputs`.

**Review by NotWoods — CHANGES_REQUESTED** [review]
I like this feature idea! I'd like to replace some code with native tools that achieve the same effect. It would also be great to have some help text above/below the JSON that says where to put it. You can use: 
>Copy this JSON into the `"icons"` array of your [Web App Manifest](https://developer.mozilla.org/en-US/docs/Web/Manifest)

**Comment by alexalannunes** [comment]
Hi @NotWoods. I'm worning on it!

**Pull request closed without merging**